database/README.md

```diff
@@ -0,0 +1,168 @@
+# Database instructions
+
+## Requirements
+
+* Kubernetes cluster
+  
+  Follow the instructions to deploy a Kubernetes cluster from [Local Deployment]([kubernetes](https://kubernetes.io/docs/tasks/tools/#kubectl)).
+* go
+  
+  [Download and install go](https://go.dev/doc/install)
+* psql
+  ```bash
+  sudo dnf install postgresql
+  psql --version
+  psql (PostgreSQL) 15.6
+  ```
+* DBeaver Community
+  
+  A SQL client is recommended to interact easily with the Database. 
+  [DBeaver Community](https://dbeaver.io/).
+
+## Manual deployment
+
+The PostgreSQL manual deployment consists in 2 parts: First, the deployment of the Database infrastructure and then, using a script, create a table in the Database and populate it.
+
+### Deploy the Database infrastructure
+
+* Create a ConfigMap with the Database details.
+
+In the file `postgres-cm.yaml` add the configuration parameters for the Database in the `data` section.
+```yaml
+# ...
+data:
+  POSTGRES_DB: <db_name> # postgresdb
+  POSTGRES_USER: <db_user> # admin or ps_user
+  POSTGRES_PASSWORD: <db_pass> # a secure password
+```
+Apply the ConfigMap configuration to the cluster.
+```bash
+kubectl apply -f postgres-cm.yaml -n [namespace]
+configmap/postgres-secret created
+```
+* Create a PersistentVolume. 
+
+The file `psql-pv.yaml` has the PersistentVolume definition, the `spec.capacity.storage` is set to `10Gi` by default.
+Apply the PersistentVolume definition to the cluster.
+```bash
+kubectl apply -f psql-pv.yaml -n [namespace]
+persistentvolume/postgres-volume created
+```
+* Create a PersistentVolumeClaim.
+
+The file `psql-claim.yaml` has the PersistentVolumeClaim definition. The `spec.resources.request.storage` is set to `10Gi` by default.
```

Comment:
Rather than "by default", I suggest saying "in the file" to make it clear where the value is set and can be changed if necessary.

```suggestion
The file `psql-claim.yaml` has the PersistentVolumeClaim definition. The `spec.resources.request.storage` is set to `10Gi` in the file.
```